### ISSUE TITLE
Accessibility] [Screen reader -Playground]: Incorrect role as link is defined for the tabs. [Sev2][WCAG4.1.2]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#37]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now, navigate to the ‘Playground’ link and activate it.
4.	Now, navigate to tabs (HTML, DOM, Line...) and check whether the proper role is defined or not.

**Actual:**
Incorrect role as link is defined for the tabs (HTML, DOM, Line...) in playground page.

**Expected:**
Proper role as Tab should be defined for the HTML, DOM, Line... in playground page and the user should be able to navigate between tabs using arrow keys.

**User Impact:**
When interactive elements have an incorrect role, it is difficult for people using assistive technology (like screen readers) to understand the purpose of the control. Knowing the role for a control helps users know what will happen when that object has focus and is selected, as well as what keyboard shortcuts to use when interacting with the control. 

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

https://user-images.githubusercontent.com/88436727/150644957-a992dcaa-e19f-45dc-b3de-803594817540.mp4
